### DISCUSSION TITLE
NativeBase select component. Can we use web look. Please read details

### DISCUSSION THREAD

**Discussion by elmcapp** [GeekyAnts/NativeBase#5108] [General]
I am using React Native for macOS. I am using the NativeBase library which works very good. I am using the select component however it is rendering a sliding sheet. I am hoping to use the looks from the web view as if you was on Mac. 

Looks I get now
<img width="406" alt="Screen Shot 2022-06-26 at 11 41 12 PM" src="https://user-images.githubusercontent.com/64275658/175855859-f9dbda26-0ad3-4a0f-85ee-4b5d483b2c9b.png">

<img width="409" alt="Screen Shot 2022-06-26 at 11 41 21 PM" src="https://user-images.githubusercontent.com/64275658/175855868-695e3d58-2f21-46d2-bbc5-0555d9538a91.png">

Here is the pop up window I would like which you get when using the Mac web view

<img width="252" alt="Screen Shot 2022-06-26 at 11 41 44 PM" src="https://user-images.githubusercontent.com/64275658/175855926-6df33f08-0d14-460b-85cb-bca8d3c9572e.png">

**Comment by elmcapp** [comment]
Looks like this is happening because it uses ActionSheet when not web view. It would be nice in your next update to provide options that is not action sheet maybe something like a dialog or modal

**Reply by sanna231** [reply]
Thanks for your suggestion. We will add this in our sprint and let you know when it's released.
